src/test/java/com/telefonica/iot/cygnus/interceptors/DestinationExtractorTest.java

```diff
@@ -1,12 +1,12 @@
 /**
- * Copyright 2014 Telefonica Investigación y Desarrollo, S.A.U
+ * Copyright 2015 Telefonica Investigación y Desarrollo, S.A.U
  *
  * This file is part of fiware-cygnus (FI-WARE project).
  *
- * cosmos-injector is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
+ * fiware-cygnus is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General
```

Comment:
Yes, the problem is sometimes it is "fiware-connectors", other time it is "orion-connectors", or "cosmos-injector"... I think in the past we played with a lot of different names before choosing "cygnus" :)